README.md

```diff
@@ -10,6 +14,7 @@ Optional prerequisites can be skipped for reduced functionality.
 -   Python >= 3.6
 -   PostgreSQL with the PostGIS extension installed
 -   An AWS S3 bucket with read/write privileges
+-   System dependencies to install the [psycopg](https://www.psycopg.org/docs/install.html#install-from-source) package
```

Comment:
you say psycopg here but psycopg2 later - should probably just be the same in both places